src/Components/PeriodicTable.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./PeriodicTable.css";
+import data from "../PeriodicTableJSON.json";
+
+
+  const colorMap = {
+    "actinide": "#7377FA",
+    "noble gas": "#FFBC42",
+    "alkaline earth metal": "#EC674E",
+    "diatomic nonmetal": "#D81159",
+    "alkali metal": "#8F2D56",
+    "transition metal": "#58586B",
+    "post-transition metal": "#218380",
+    "polyatomic nonmetal": "#40FF4D",
+    lanthanide: "#4AABAF",
+    metalloid: "#73D2DE",
+    
+  };
```

Comment:
I think you shouldn't use whitespaces as a key 🤔 
```suggestion
  const colorMap = {
    "actinide": "#7377FA",
    "noble-gas": "#FFBC42",
    "alkaline-earth-metal": "#EC674E",
    "diatomic-nonmetal": "#D81159",
    "alkali-metal": "#8F2D56",
    "transition-metal": "#58586B",
    "post-transition-metal": "#218380",
    "polyatomic-nonmetal": "#40FF4D",
    "lanthanide": "#4AABAF",
    "metalloid": "#73D2DE",
  };
```

Reply:
makes sense. Thank you! ✌🏼